task-list/app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,53 @@
+class TasksController < ApplicationController
+
+
+  def index   #this is enough for the controller to render a view, but need to tell rails when this is an appropriate action
+    @all_tasks = all_incomplete_tasks
```

Comment:
I'd probably name your instance variable something more specific than @all_tasks (unless there's a good reason why you're doing it this way). Maybe @all_incomplete_tasks, since that's what it represents?